src/main/java/org/gitlab4j/api/models/HealthCheckStatusLabel.java

```diff
@@ -0,0 +1,17 @@
+package org.gitlab4j.api.models;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+public class HealthCheckStatusLabel {
+    private String shard;
```

Comment:
The only extra label I had was "shard". If you have an idea of other labels, I am willing to add them.

Ideally, this should be an enum as well but I have not encountered other values while testing.